api/src/resolvers/utils/notify.ts

```diff
@@ -42,7 +61,7 @@ export const sendBulkSMS = async (recipient: string[], message: string) => {
       'content-type': 'application/json',
     },
     data: {
-      recipient,
+      recipient: [process.env.TEST_PHONE_NUMBER] || recipient,
```

Comment:
I feel like this looks like it will bring a logic error.  an empty array is still a truthy value in javascript. 

Let's replace it with some conditional logic to be sure.

`process.env.TEST_PHONE_NUMBER ? [process.env.TEST_PHONE_NUMBER] : recipient,`